paasta_tools/setup_prometheus_adapter_config.py

```diff
@@ -196,6 +196,22 @@ def should_create_uwsgi_scaling_rule(
     return False, "did not request uwsgi autoscaling"
 
 
+def should_create_piscina_scaling_rule(
+    autoscaling_config: AutoscalingParamsDict,
+) -> Tuple[bool, Optional[str]]:
+    """
+    Determines whether we should configure the prometheus adapter for a given service.
+    Returns a 2-tuple of (should_create, reason_to_skip)
+    """
+    if autoscaling_config["metrics_provider"] == "piscina":
+        if not autoscaling_config.get("use_prometheus", DEFAULT_USE_PROMETHEUS_UWSGI):
+            return False, "requested piscina autoscaling, but not using Prometheus"
+
+        return True, None
+
+    return False, "did not request piscina autoscaling"
```

Comment:
same as earlier in this PR - we can probably skip teaching this new code about `use_prometheus`
```suggestion
    if autoscaling_config["metrics_provider"] == "piscina":
        return True, None
    return False, "did not request piscina autoscaling"
```